less/mixins.less

```diff
@@ -1 +1,48 @@
-// Mixins in here
\ No newline at end of file
+// Mixins in here
+.link-reset(@color) {
+  text-decoration: none;
+  list-style-type: none;
+  color: @color;
+}
+
+@desktop: ~"(min-width: 800px)";
+
+.flexy(
+  @disp: flex,
```

Comment:
You might want to put this variable in your `variables.less` file for easier tracking of your media query size settings..